customer_service/lib/FileManager/FileCreator/FileCreator.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace Lib\FileManager\FileCreator;
+
+class FileCreator
+{
+    private $id;
+    private $name;
+    private $description;
+    private $fileDestinationPath;
+    private $createdTime;
+    private $lastUpdate;
+    private $status;
+
+    public function __construct($id, $name, $description, $fileDestinationPath, $createdTime, $lastUpdateTime, $status)
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->description = $description;
+        $this->fileDestinationPath = $fileDestinationPath;
+        $this->createdTime = $createdTime;
+        $this->lastUpdateTime = $lastUpdateTime;
+        $this->status = $status;
+    }
+
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    public function setId($id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)